archetypes/oci-mp/src/main/resources/assembly/distribution.xml

```diff
@@ -0,0 +1,18 @@
+<assembly xmlns="http://maven.apache.org/ASSEMBLY/2.1.0"
```

Comment:
Is this still required ? The other archetypes do not provide an assembly. We should either file an issue to add that to all archetypes or remove it.

Reply:
Moved under server as discussed.